ChangeLog

```diff
@@ -1,5 +1,15 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+API changes
```

Comment:
Can we rename to "API extensions" of something else that reflects that the API remains compatible, as is our policy? I don't want to mislead people into being afraid of incompatibilities while we actually do the hard work of paying attention to compatibility.

Reply:
I personally agree, but it's out of scope of this PR, we'll discuss it internally.